Polyfills/URL/Source/URL.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <Babylon/JsRuntimeScheduler.h>
+
+#include <napi/napi.h>
+#include "URLSearchParams.h"
+
+namespace Babylon::Polyfills::Internal
+{
+    class URL final : public Napi::ObjectWrap<URL>
+    {
+        static constexpr auto JS_URL_CONSTRUCTOR_NAME = "URL";
+
+    public:
+        static void Initialize(Napi::Env env);
+        static URL& GetFromJavaScript(Napi::Env env);
+
+        explicit URL(const Napi::CallbackInfo& info);
+
+    private:
+
```

Comment:
nit: extra newline (inconsistent with other code)

```suggestion
    private:
```